templates/cloudflare-workers/wrangler.toml

```diff
@@ -1,17 +1,13 @@
 name = "remix-cloudflare-workers"
-type = "javascript"
 
-zone_id = ""
+compatibility_date = "2022-05-11"
+
```

Comment:
I think this one can be omitted, no?

```suggestion
```

Reply:
Specifying `compatibility_date` is required.
```
> build
> remix build

Building Remix app in production mode...
Built in 370ms
 ⛅️ wrangler 2.0.2
-------------------
Running custom build: npm run build

> build
> remix build

Building Remix app in production mode...
Built in 286ms

✘ [ERROR] A compatibility_date is required when publishing. Add one to your wrangler.toml file, or pass it in your terminal as --compatibility-date. See https://developers.cloudflare.com/workers/platform/compatibility-dates for more information.


If you think this is a bug then please create an issue at https://github.com/cloudflare/wrangler2/issues/new.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```